appdaemontestframework/init_framework.py

```diff
@@ -111,7 +111,7 @@ def log_log(msg, level='INFO'):
 
 def _mock_hass_init(hass_functions):
     """Mock the Hass object init and set up class attributes that are used by automations"""
-    def hass_init_mock(self, _ad, name, _logger, _error, _args, _config, _app_config, _global_vars):
+    def hass_init_mock(self, _ad, name, _logger, _error, _args, _config, _app_config):
```

Comment:
It looks like this has been fixed on `master` since you branched off since the refactor that replaced `init_framework.py` with `hass_mocks.py`. Can you merge/rebase `master` and see if that's the case.

Reply:
➕1 
This should be fixed already 👍